### ISSUE TITLE
FocusCropHeight appears to be broken after upgrade to 4.0

### ISSUE THREAD

**Issue by davejtoews** [jonom/silverstripe-focuspoint#98]
Attempting to use some code that worked properly in 3.x I end up with the following error in 4.0

```
[Emergency] Uncaught Error: Call to a member function getHeight() on array
GET /invest?flush

Line 98 in /var/www/html/vendor/jonom/focuspoint/src/FieldType/DBFocusPoint.php
Source

89      * @return int
90      */
91     public function getHeight(): int
92     {
93         $height = $this->getField('Height');
94         if ($height) {
95             return intval($height);
96         }
97         if ($this->record) {
98             return intval($this->record->getHeight());
99         }
100         return 0;
101     }
102 
103     /**
104      * Set the focus point Y coordinate
Trace

JonoM\FocusPoint\FieldType\DBFocusPoint->getHeight() 
ViewableData.php:133
SilverStripe\View\ViewableData->__get(Height) 
DBComposite.php:70
SilverStripe\ORM\FieldType\DBComposite->__get(Height) 
FocusPointExtension.php:81
JonoM\FocusPoint\Extensions\FocusPointExtension->FocusCropHeight(468) 
call_user_func_array(Array, Array) 
Extensible.php:144
SilverStripe\View\ViewableData->SilverStripe\Core\{closure}(<img src="/assets/Uploads/igor-kyryliuk-ihYFBetJY98-unsplash__Resampled.png" alt="igor kyryliuk ihYFBetJY98 unsplash" /> , Array) 
CustomMethods.php:61
SilverStripe\View\ViewableData->__call(FocusCropHeight, Array) 
LazyFocusFitImageExtension.php:141
```

**Comment by eehondas** [comment]
+1 on this. Exact same error.

**Comment by lerni** [comment]
@micschk with this merged (4.0.2) now we see `[Notice] Undefined index: Width` instead of `[Emergency] Uncaught Error: Call to a member function getWidth() on array`

I suggest https://github.com/jonom/silverstripe-focuspoint/pull/103

Line 80
```diff
- return intval(is_array($this->record) ? $this->record["Width"] : $this->record->getWidth());
+ return intval(is_array($this->record) ? array_key_exists("Width",$this->record) : $this->record->getWidth());
```

Line 98
```diff
- return intval(is_array($this->record) ? $this->record["Height"] : $this->record->getHeight());
+ return intval(is_array($this->record) ? array_key_exists("Height",$this->record) : $this->record->getHeight());
```
